### PR TITLE
feat: added the resources for learning git

### LINKED ISSUE #2030: [ADD] new link for git learning resources
### Category Name

Other

### Subcategory Name

Git

### Additional Context

Aim to introduce a new sub-category called Git which contains the resources to learn git commands. 

### Checklist

- [X] I have checked the existing [issues](https://github.com/rupali-codes/LinksHub/issues?q=is%3Aissue+)
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I have checked that this category does not exist
- [X] I have checked that this subcategory does not exist
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by Sanchitbajaj02**
## Fixes Issue

closes #2030 

## Changes proposed

Added 3 resources to learn about git

### CODE REVIEW

File: database/other/git.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  {
+    "name": "Learn Git Branching",
+    "description": "This website gives you an interactive tool to learn about the git commands.",
+    "url": "https://learngitbranching.js.org/",
+    "category": "other",
+    "subcategory": "git"
+  },
+  {
+    "name": "Git Book",
+    "description": "It is one of the great books to learn git. It contains expert Git guide with tutorials, code examples, and collaboration tips.",
```
Comment (CBID2) [inline database/other/git.json:11]:
```suggestion
    "description": "This is one of the great books to learn Git. It contains tutorials, code examples, and collaboration tips.",
```

File: database/other/git.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "url": "https://git-scm.com/book/en/v2",
+    "category": "other",
+    "subcategory": "git"
+  },
+  {
+    "name": "Version Control with Git",
+    "description": "It is a free course provided by 'Udacity' which covers the essentials of using version control system using git.",
```
Comment (CBID2) [inline database/other/git.json:18]:
```suggestion
    "description": "It is a free course provided by Udacity, which covers the essentials of using Git.",
```

**Review by rupali-codes — APPROVED** [review]
looks good to me, but from next time, please make sure to keep your changes only related to your issue, don't overdo anything :)

**Review by CBID2 — CHANGES_REQUESTED** [review]
Made some suggestions to improve your descriptions. Other than that, you're off to a great start! :)

File: database/other/git.json (hunk 1 above)
Comment (rupali-codes) [reply database/other/git.json:11]: i guess it looks fine

File: database/other/git.json (hunk 2 above)
Comment (rupali-codes) [reply database/other/git.json:18]: also this

**Comment by Sanchitbajaj02** [comment]
> looks good to me, but from next time, please make sure to keep your changes only related to your issue, don't overdo anything :)

Sure I will keep that in mind 😊

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
@Sanchitbajaj02 
Please add a description.

image

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Review by rupali-codes — APPROVED** [review]
Lgtm

**Pull request merged**